src/components/pricing/TierBulletsSection.js

```diff
@@ -1,14 +1,22 @@
 import React from 'react';
-import { CheckIcon } from '@heroicons/react/solid'
+import { CheckIcon, PlusIcon } from '@heroicons/react/solid';
 
-const TierBullet = ({ feature, hasIcon = true }) => (
+const renderIcon = ({ iconName }) => {
```

Comment:
Is there a more "React" way to dynamically choose an icon like this?

Reply:
You'd make this `renderIcon` function into a component that takes a prop of the icon name. For typescript you also want to restrict the available icon names to the actually supported strings.